### PR TITLE
* Fix Sanitize Argument null exception.

### PR DESCRIPTION
**Pull request by gmaf**

### CODE REVIEW

File: src/Lykke.Logs/Loggers/LykkeSanitizing/SanitizingLog.cs [hunk 1]
```diff
@@ -26,7 +26,12 @@ public ISanitizingLog AddSanitizingFilter(Regex pattern, string replacement)
             return this;
         }
 
-        public string Sanitize(string value) => _options.Filters.Aggregate(value, (a, p) => p.Pattern.Replace(a, p.Replacement));
+        public string Sanitize(string value)
+        {
+            return string.IsNullOrWhiteSpace(value)
```
Comment (KonstantinRyazantsev) [inline src/Lykke.Logs/Loggers/LykkeSanitizing/SanitizingLog.cs:31]: If value is null, better to return null, otherwise it could break something

Comment (gmaf) [reply src/Lykke.Logs/Loggers/LykkeSanitizing/SanitizingLog.cs:31]: Done, now return value.

File: tests/Lykke.Logs.Tests/SanitizingLogTests.cs [hunk 2]
```diff
@@ -126,5 +127,34 @@ public void ShouldConfigureOptions()
...
+        {
+            // Arrange
+            var fakeLog = Substitute.For<ILog>();
+            var sanitizer = new SanitizingLog(fakeLog, new SanitizingOptions());
+
+            // Act
+            var result = sanitizer.Sanitize(null);
+
+            // Assert
+            Assert.Equal(null, result);
```
Comment (starkmsu) [inline tests/Lykke.Logs.Tests/SanitizingLogTests.cs]: Asser.Null(result) could be used here

File: tests/Lykke.Logs.Tests/SanitizingLogTests.cs [hunk 3]
```diff
@@ -126,5 +127,34 @@ public void ShouldConfigureOptions()
...
+            // Arrange
+            var fakeLog = Substitute.For<ILog>();
+            var sanitizer = new SanitizingLog(fakeLog, new SanitizingOptions());
+            sanitizer.AddSanitizingFilter(new Regex(""), "");
+
+            // Act
+            var result = sanitizer.Sanitize(null);
+
+            // Assert
+            Assert.Equal(null, result);
```
Comment (starkmsu) [inline tests/Lykke.Logs.Tests/SanitizingLogTests.cs]: Same as above

**Review by starkmsu — APPROVED** [review]

File: tests/Lykke.Logs.Tests/SanitizingLogTests.cs (hunk 2 above)
Comment (gmaf) [reply tests/Lykke.Logs.Tests/SanitizingLogTests.cs]: Fixed.

File: tests/Lykke.Logs.Tests/SanitizingLogTests.cs (hunk 3 above)
Comment (gmaf) [reply tests/Lykke.Logs.Tests/SanitizingLogTests.cs]: Fixed.

**Review by KonstantinRyazantsev — APPROVED** [review]

**Pull request merged**